lib/Rx/Operator/RetryOperator.php

```diff
@@ -41,7 +41,9 @@ function ($error) use ($observable, $observer, &$disposable, &$getNewObserver, $
                         $observer->onError($error);
                         return;
                     }
-                    $disposable->dispose();
+                    if ($disposable) {
```

Comment:
Sorry, I missed this comment.  I'll change it to a SerializeDisposable, so it's clear what's going on.